nvaccess/nvda
# Removed unuseful empty or English-only documentation locale files

Removed unuseful documentation files for the following languages:
- Kanada (kn) change log and user guide - Punjabi (pa) change log and user guide
- Hebrew (he) change log only
- Irish (ga) change log only
- Icelandic (is) user guide
- 
This files are either almost empty or contain only English content. Removing them allows to users of these languages to open up-to-date English documentation instead of empty or outdated English documentation.

### Link to issue description
See discussion at:
https://groups.io/g/nvda-translations/topic/removal_of_untranslated_or/94536263?p=,,,100,0,0,0::recentpostdate/sticky,,,100,2,0,94536263,previd%3D1668500630019398565,nextid%3D1637210175951877387&previd=1668500630019398565&nextid=1637210175951877387

And also:
https://groups.io/g/nvda-translations/topic/status_of_icelandic_is_user/94537010?p=,,,100,0,0,0::recentpostdate/sticky,,,100,2,0,94537010,previd%3D1668500630019398565,nextid%3D1637210175951877387&previd=1668500630019398565&nextid=1637210175951877387

### Summary of the issue:

Some locale documentation files (change log or user guide) are empty or contain only out-of-date English text:
- Kanada (kn) change log and user guide
- Punjabi (pa) change log and user guide
- Hebrew (he) change log only
- Irish (ga) change log only
- Icelandic (is) user guide only

Since there is no content written in locale language in these files, the users of these languages have an empty documentation or an out-of-date English documentation.

If there is not translation work on these files for now (and there has actually never been except for Icelandic), it would be better for the users to have at least the up-to-date English documentation.

### Description of user facing changes
Users will have up-to-date English documentation files instead of empty or out-of-date English-only documentation files.

### Description of development approach
Remove English-only or empty files in the locale doc folders.

### Testing strategy:
Manual tests:
Run NVDA in these languages and check that up-to-date documentation is accessible from NVDA menu.

### Known issues with pull request:
* There are other unmaintained and out-of-date documentation files for some locales. If unmaintained files contain text in the locale language, this PR does not remove them. The question of removing out-of-date translation work is too broad for this PR and would require a contact with an active community of the impacted languages which is often difficult.
* Translation freeze for NVDA 2022.4 has just begun today. In case translators want to translate the files removed in this PR for 2022.4 (what is very unlikely given the [linked messages](https://groups.io/g/nvda-translations/topic/removal_of_untranslated_or/95040422?p=,,,100,0,0,0::recentpostdate/sticky,,,100,2,0,95040422,previd%3D1668511486170139502,nextid%3D1637401426641644297&previd=1668511486170139502&nextid=1637401426641644297) on the translators mailing list), care should be taken not to override their work merging this PR. I will keep an eye on SVN as well as on translation mailing list and inform here if new translation work occurs for these files.

### Change log entries:
Not needed
### Code Review Checklist:
  - description is up to date
  - change log entries
  - Unit tests
  - System (end to end) tests
  - Manual testing
  - User Documentation
  - Developer / Technical Documentation
  - Context sensitive help for GUI changes
  - Speech 
  - Braille
  - Low Vision
  - Different web browsers
  - Localization in other languages / culture than English
